db/models/ally.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Ally = db.define('allies', {
+  username: Sequelize.STRING,
+  phone_number: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  email: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      isEmail: true
+    }
+  },
+  job_title: Sequelize.STRING,
+  company: Sequelize.STRING,
+  industry: Sequelize.ARRAY(Sequelize.STRING),
+  max_clients: Sequelize.INTEGER
```

Comment:
Oh, is this something they input when they sign up saying "I'm willing to do x number"?

(That would make a lot more sense. I was for some reason thinking we were imposing a limit on them?)